src/pages/dashboard/nodeSignIn/NodeSignInPSPForm.tsx

```diff
@@ -1,411 +1,396 @@
+                digitalStamp:
+                    values.digitalStamp === undefined
+                        ? t('nodeSignInPage.validationMessage.requiredField')
+                        : undefined,
+            }).filter(([_key, value]) => value)
+        );
 
-  const handleChangeNumberOnly = (
-    e: React.ChangeEvent<any>,
-    field: string,
-    formik: FormikProps<NodeOnSignInPSP>
-  ) => {
-    const regex = /^[A-Z0-9]+$/;
-    if (e.target.value === '' || regex.test(e.target.value)) {
-      formik.setFieldValue(field, e.target.value);
-    }
-  };
+    const handleChangeNumberOnly = (
+        e: React.ChangeEvent<any>,
+        field: string,
+        formik: FormikProps<NodeOnSignInPSP>
+    ) => {
+        const regex = /^[A-Z0-9]+$/;
+        if (e.target.value === '' || regex.test(e.target.value)) {
+            formik.setFieldValue(field, e.target.value);
+        }
+    };
 
-  const formik = useFormik<NodeOnSignInPSP>({
-    initialValues: initialFormData(selectedParty),
-    validate,
-    onSubmit: async () => {
-      await submit();
-    },
-    enableReinitialize: true,
-  });
+    const formik = useFormik<NodeOnSignInPSP>({
+        initialValues: initialFormData(selectedParty),
+        validate,
+        onSubmit: async () => {
+            await submit();
+        },
+        enableReinitialize: true,
+    });
 
-  useEffect(() => {
-    if (
-      !signInData?.paymentServiceProviderDetailsResource?.psp_code &&
-      !/^\d{5}$/.test(selectedParty?.pspData?.abi_code ?? '')
-    ) {
-      // eslint-disable-next-line @typescript-eslint/no-floating-promises
-      formik.setFieldValue('pspCode', formik.values.bicCode);
-    }
-  }, [formik.values.bicCode]);
+    useEffect(() => {
+        if (
+            !signInData?.paymentServiceProviderDetailsResource?.psp_code &&
+            !/^\d{5}$/.test(selectedParty?.pspData?.abi_code ?? '')
+        ) {
+            // eslint-disable-next-line @typescript-eslint/no-floating-promises
+            formik.setFieldValue('pspCode', formik.values.bicCode);
+        }
+    }, [formik.values.bicCode]);
 
-  // eslint-disable-next-line sonarjs/cognitive-complexity
-  const submit = async () => {
-    setLoading(true);
+    // eslint-disable-next-line sonarjs/cognitive-complexity
+    const submit = async () => {
+        setLoading(true);
 
-    if (selectedParty && isPspSigned(signInData)) {
-      try {
-        if (!isPspBrokerSigned(signInData) && intermediaryAvailableValue) {
-          await createPspBroker({
-            broker_psp_code: selectedParty.fiscalCode,
-            description: selectedParty.description,
-            enabled: true,
-            extended_fault_bean: true,
-          });
-        }
+        if (selectedParty && orgInfo.isSigned && orgInfo.types.isPsp) {
+            try {
+                if (!orgIsPspDirect && intermediaryAvailableValue) {
```

Comment:
At line 58 it was `pspDirect` and now is `orgIsPspDirect` (line 43), at line 166 it was `isPspBrokerSigned ` and now is `orgIsPspDirect` (line 151). 
It is correct to use the same boolean for what was before two different variables?